fineract-provider/src/main/java/org/apache/fineract/batch/command/internal/AdjustLoanTransactionCommandStrategy.java

```diff
@@ -44,7 +44,7 @@
  */
 @Component
 @RequiredArgsConstructor
-public class AdjustTransactionCommandStrategy implements CommandStrategy {
+public class AdjustLoanTransactionCommandStrategy implements CommandStrategy {
 
     /**
      * Loan transactions api resource {@link LoanTransactionsApiResource}.
```

Comment:
Arent these changes irrelevant to [FINERACT-1921](https://issues.apache.org/jira/browse/FINERACT-1921)? Please extract them into a separate PR if so!